### DISCUSSION TITLE
How to classify usage tokens/usage cost by meta data

### DISCUSSION THREAD

**Discussion by aravinth-numberz** [openlit/openlit#435] [Q&A]
I have use case where i need to send alert based on the usage of tokens by user, so i'm passing user id in metadata so i can check the logs by user id. But the usage tokens and usage cost calculation logs coming as a individual log in elastic(I'm using elastic and openlit combined) couldn't use them to create a data such as user based usage tokens.

**Comment by amanagarwal042** [comment]
Hey @aravinth-numberz , Qq are you using the [Manual Tracing](https://docs.openlit.io/latest/features/tracing#manual-tracing) in OpenLIT to additionally add user id?
Want to get an understanding of your setup before I can help with a solution
